app/main.py

```diff
@@ -1,3 +1,54 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: "Distance") -> "Distance":
```

Comment:
Indicate that `other` could be `int` or `float` in the type annotation as well using `|`. Do not use `"` in the type annotation, indicate class directly. Apply similar changes in the other methods as well. In case you face errors, use `from __future__ import annotations` import